BedrockConflictManager.cpp

```diff
@@ -0,0 +1,63 @@
+#include "BedrockConflictManager.h"
+#include <libstuff/libstuff.h>
+
+BedrockConflictManager::BedrockConflictManager() {
+}
+
+void BedrockConflictManager::recordTables(const string& commandName, const set<string>& tables) {
+    {
+        lock_guard<mutex> lock(m);
+        auto commandInfo = _commandInfo.find(commandName);
+        if (commandInfo == _commandInfo.end()) {
+            commandInfo = _commandInfo.emplace(make_pair(commandName, BedrockConflictManagerCommandInfo())).first;
+        }
+
+        // Increase the count of the command in general.
+        commandInfo->second.count++;
+
+        // And for each table (that's not a journal).
```

Comment:
Hmmm, but commands conflict on journal tables though...

Reply:
Would've loved if this comment was in the code so I wouldn't have to go hunting for it till I found it here